package.json

```diff
@@ -31,10 +31,23 @@
   ],
   "devDependencies": {
     "chai": "^3.5.0",
+    "chai-as-promised": "^6.0.0",
     "eslint": "^3.2.2",
     "eslint-config-screwdriver": "^2.0.0",
-    "eslint-plugin-import": "^1.12.0",
-    "jenkins-mocha": "^2.6.0"
+    "eslint-plugin-import": "^2.0.0",
```

Comment:
This will cause a peerDependency warning with eslint-config-airbnb-base until they update their peerDeps.